contents/speakers.json

```diff
@@ -0,0 +1,53 @@
+{
+  "template": "pages/speakers.html.njk",
+  "speakers": [
```

Comment:
More I think about it, more I lean towards using `contents/speakers/SPEAKER-NAME.md` approach as we do right now for JSConf. I will re-factor it.